.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "deno.enable": true,
+  "deno.lint": true,
+  "deno.unstable": false,
+  "deno.config": "./tsconfig.json",
+}
```

Comment:
I prefer to keep the project editor-agnostic.

Reply:
No problem. Will have to be removed.